docker-compose-conflictmonitor.yml

```
@@ -0,0 +1,158 @@
+services:
+  conflictmonitor:
+    profiles:
+      - conflictmonitor
+    image: usdotjpoode/jpo-conflictmonitor:latest
+    restart: ${RESTART_POLICY}
+    ports:
+      - '8082:8082'
+    environment:
+      DOCKER_HOST_IP: ${DOCKER_HOST_IP:?error}
+      KAFKA_BOOTSTRAP_SERVERS: ${KAFKA_BOOTSTRAP_SERVERS:?error}
+    healthcheck:
+      test: ['CMD', 'java', '-version']
+      interval: 10s
+      timeout: 10s
+      retries: 20
+    logging:
+      options:
+        max-size: '10m'
+        max-file: '5'
+    deploy:
+      resources:
+        limits:
+          memory: 3G
+    depends_on:
+      kafka:
+        condition: service_healthy
+        required: false
+
+  ode:
+    profiles:
+      - conflictmonitor
+    image: usdotjpoode/jpo-ode:latest
+    restart: ${RESTART_POLICY}
+    ports:
+      - '8080:8080'
+      - '9090:9090'
+      - '46753:46753/udp'
+      - '46800:46800/udp'
+      - '47900:47900/udp'
+      - '44900:44900/udp'
+      - '44910:44910/udp'
+      - '44920:44920/udp'
+      - '44930:44930/udp'
+      - '44940:44940/udp'
+      - '44990:44990/udp'
+      - '5555:5555/udp'
+      - '6666:6666/udp'
+    environment:
+      DOCKER_HOST_IP: ${DOCKER_HOST_IP:?error}
```

Issue (blocking): The latest version of the ODE will not work properly without the following environment variables:
```suggestion
    environment:
      DOCKER_HOST_IP: ${DOCKER_HOST_IP:?error}
      ODE_TIM_INGEST_MONITORING_ENABLED: true
      ODE_TIM_INGEST_MONITORING_INTERVAL: 60
      DATA_SIGNING_ENABLED_RSU: false
      DATA_SIGNING_ENABLED_SDW: false
      ODE_KAFKA_BROKERS: ${DOCKER_HOST_IP}:9092
      KAFKA_LINGER_MS: 1
      KAFKA_BATCH_SIZE: 16384
      KAFKA_RETRIES: 0
```